addons/mrp/models/mrp_workcenter.py

```diff
@@ -32,7 +32,11 @@ class MrpWorkcenter(models.Model):
         help="Description of the Work Center.")
     default_capacity = fields.Float(
         'Capacity', default=1.0,
+<<<<<<< HEAD
         help="Default number of pieces that can be produced in parallel. In case the work center has a capacity of 5 and you have to produce 10 units on your work order, the usual operation time will be multiplied by 2.")
+=======
+        help="Number of pieces (in product UoM) that can be produced in parallel (at the same time) at this work center. For example: the capacity is 5 and you need to produce 10 units, then the operation time listed on the BOM will be multiplied by two. However, note that both time before and after production will only be counted once.")
```

Comment:
This is a trickier conflict to fix. The field name changed to `default_capacity` (you will have to adjust the .pot to match the change). The help text will need to be adjusted since the purpose of this field also slightly changed by adding "Default" to the beginning of it, i.e.:

```
Default number of pieces (in product UoM) that can be produced in parallel (at the same time) at this work center.
```

(with the rest of your new text unchanged)

Reply:
I think you misunderstood me. You should copy the entire help text from the original cherry-picked commit, but add "Default" in the front of it. So it should be

```
Default number of pieces (in product UoM) that can be produced in parallel (at the same time) at this work center. For example: the capacity is 5 and you need to produce 10 units, then the operation time listed on the BOM will be multiplied by two. However, note that both time before and after production will only be counted once.
```

Reply:
@ticodoo done